webview/src/experiments/components/table/styles.module.scss

```diff
@@ -258,6 +258,8 @@ $bullet-size: calc(var(--design-unit) * 4px);
 
   .tableContainer {
     flex: 1;
+    display: flex;
```

Comment:
The checkboxes are unchecking anytime a user clicks outside of the `table` element. I don't think this should be happening but I couldn't figure out why it's happening in the small amount of time I chose to look at the issue 🤔

Reply:
I think the issue is found in the following code
```

  const clickOutsideHandler = useCallback(() => {
    clearSelectedRows?.()
  }, [clearSelectedRows])

  useClickOutside(tableRef, clickOutsideHandler)
```
On line 45 of file `webview/src/experiments/components/Table.tsx`

Reply:
Could probably remove the whole `useClickOutside`.